state_synchronizer/src/coordinator.rs

```diff
@@ -393,7 +409,11 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                 let mut msg = StateSynchronizerMsg::new();
                 msg.set_chunk_request(req);
 
-                if sender.send_to(peer_id, msg).await.is_err() {
+                if let Err(e) = sender.send_to(peer_id, msg).await {
+                    if e.kind() == NetworkErrorKind::TimedOut {
```

Comment:
we want a bit different timeout behavior. Timeout penalty would be here to await response for chunk from peer and if it's not there in expected time, decrease score